dvc/config.py

```diff
@@ -238,12 +239,11 @@ def __init__(self, dvc_dir=None, validate=True):
     def get_dir(cls, level):
         from appdirs import user_config_dir, site_config_dir
 
-        assert level in ("global", "system")
```

Comment:
These changes are required by deepsource

Reply:
@karajan1001 DS is a bit strict on asserts in code, we just ignore those complaints.